packages/sdk/src/authentication/iAuthProvider.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AxiosRequestConfig } from "axios";
+
+// Defines method that injects authentication info to http requests
+export interface IAuthProvider {
```

Comment:
I happened to find that ts does not recommend use `I` as interface prefix. Let's just use `AuthProvider` as interface name.
https://stackoverflow.com/questions/31876947/confused-about-the-interface-and-class-coding-guidelines-for-typescript

Reply:
Fixed.